src/test/operators/join_full_test.cpp

```diff
@@ -250,6 +250,13 @@ TYPED_TEST(JoinFullTest, SmallerOuterJoin) {
                                              "resources/test_data/tbl/joinoperators/int_smaller_outer_join.tbl", 1);
 }
 
+TYPED_TEST(JoinFullTest, SmallerOuterJoinWithNull) {
```

Comment:
Any reason you're specifically testing the outer join?

Reply:
Could also be left or right join. But only these three materialize NULL values.